diff --git a/src/Mikkelhm.Core/Mikkelhm.Core.csproj b/src/Mikkelhm.Core/Mikkelhm.Core.csproj
index 14f16a7..ad42d7f 100644
--- a/src/Mikkelhm.Core/Mikkelhm.Core.csproj
+++ b/src/Mikkelhm.Core/Mikkelhm.Core.csproj
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Umbraco.Cms.Core" Version="12.3.3" />
+    <PackageReference Include="Umbraco.Cms.Core" Version="12.3.5" />
     <PackageReference Include="Microsoft.AspNetCore.Html.Abstractions" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.ViewFeatures" Version="2.2.0" />
   </ItemGroup>
diff --git a/src/Mikkelhm.Web/Mikkelhm.Web.csproj b/src/Mikkelhm.Web/Mikkelhm.Web.csproj
index 48a3e28..959cb19 100644
--- a/src/Mikkelhm.Web/Mikkelhm.Web.csproj
+++ b/src/Mikkelhm.Web/Mikkelhm.Web.csproj
@@ -9,11 +9,11 @@
     <PackageReference Include="Umbraco.Cloud.Cms.PublicAccess" Version="12.0.0" />
     <PackageReference Include="Umbraco.Cloud.Identity.Cms" Version="12.0.32" />
     <PackageReference Include="Umbraco.Cloud.StorageProviders.AzureBlob" Version="12.0.0" />
-    <PackageReference Include="Umbraco.Cms" Version="12.3.3" />
+    <PackageReference Include="Umbraco.Cms" Version="12.3.5" />
     <PackageReference Include="Umbraco.Deploy.Cloud" Version="12.0.5" />
     <PackageReference Include="Umbraco.Deploy.Contrib" Version="12.0.0" />
     <PackageReference Include="Umbraco.Deploy.Forms" Version="12.0.5" />
-    <PackageReference Include="Umbraco.Forms" Version="12.1.2" />
+    <PackageReference Include="Umbraco.Forms" Version="12.2.0" />
   </ItemGroup>
 
   <ItemGroup>
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type-container__32c8376907fc425e8619a8a01985ae4f.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type-container__32c8376907fc425e8619a8a01985ae4f.uda
index ca4412f..40f523c 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type-container__32c8376907fc425e8619a8a01985ae4f.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type-container__32c8376907fc425e8619a8a01985ae4f.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://data-type-container/32c8376907fc425e8619a8a01985ae4f",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__159ea2ee26334d2e8fd188c634159f70.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__159ea2ee26334d2e8fd188c634159f70.uda
index afdba57..abbf908 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__159ea2ee26334d2e8fd188c634159f70.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__159ea2ee26334d2e8fd188c634159f70.uda
@@ -16,5 +16,5 @@
   "Udi": "umb://data-type/159ea2ee26334d2e8fd188c634159f70",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__265ba68c5ab1491a918edab269e7334e.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__265ba68c5ab1491a918edab269e7334e.uda
index 04abd09..921e451 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__265ba68c5ab1491a918edab269e7334e.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__265ba68c5ab1491a918edab269e7334e.uda
@@ -47,5 +47,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__26fdbe41a0c34468a9b4f468f3c2cea1.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__26fdbe41a0c34468a9b4f468f3c2cea1.uda
index 3c7c07c..e9cbc0d 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__26fdbe41a0c34468a9b4f468f3c2cea1.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__26fdbe41a0c34468a9b4f468f3c2cea1.uda
@@ -63,5 +63,5 @@
   "Udi": "umb://data-type/26fdbe41a0c34468a9b4f468f3c2cea1",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__55aca15649b940f885a4200c1f0e7b67.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__55aca15649b940f885a4200c1f0e7b67.uda
index 85bc6b8..745f1e4 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__55aca15649b940f885a4200c1f0e7b67.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__55aca15649b940f885a4200c1f0e7b67.uda
@@ -14,5 +14,5 @@
   "Udi": "umb://data-type/55aca15649b940f885a4200c1f0e7b67",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__8f89b8ab84d24ff0838932954b8404b6.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__8f89b8ab84d24ff0838932954b8404b6.uda
index 218b74d..db66b4d 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__8f89b8ab84d24ff0838932954b8404b6.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__8f89b8ab84d24ff0838932954b8404b6.uda
@@ -39,5 +39,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
index 55cd682..9c0488a 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__92897bc6a5f34ffeae27f2e7e33dda49.uda
@@ -11,5 +11,5 @@
   "Udi": "umb://data-type/92897bc6a5f34ffeae27f2e7e33dda49",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__9fea27f93228458699bc1e04e51bb54a.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__9fea27f93228458699bc1e04e51bb54a.uda
index 2b2d585..b9ca9bd 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__9fea27f93228458699bc1e04e51bb54a.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__9fea27f93228458699bc1e04e51bb54a.uda
@@ -55,5 +55,5 @@
   "Udi": "umb://data-type/9fea27f93228458699bc1e04e51bb54a",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__dc34b7119e5b4a6da59b79c9f08f338d.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__dc34b7119e5b4a6da59b79c9f08f338d.uda
index 2324c3e..2082196 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__dc34b7119e5b4a6da59b79c9f08f338d.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__dc34b7119e5b4a6da59b79c9f08f338d.uda
@@ -18,5 +18,5 @@
   "Udi": "umb://data-type/dc34b7119e5b4a6da59b79c9f08f338d",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__f2d7803b41604b69b1e383365bb3107c.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__f2d7803b41604b69b1e383365bb3107c.uda
index 3e0d005..248c156 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__f2d7803b41604b69b1e383365bb3107c.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/data-type__f2d7803b41604b69b1e383365bb3107c.uda
@@ -12,5 +12,5 @@
   "Udi": "umb://data-type/f2d7803b41604b69b1e383365bb3107c",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.DataTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__61464945308042199a697c71009c7941.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__61464945308042199a697c71009c7941.uda
index 4a8a6d4..6894437 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__61464945308042199a697c71009c7941.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__61464945308042199a697c71009c7941.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://document-type-container/61464945308042199a697c71009c7941",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__7cc2deafa60c41c2a18a184d1caf3e62.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__7cc2deafa60c41c2a18a184d1caf3e62.uda
index a82cf0f..05fc874 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__7cc2deafa60c41c2a18a184d1caf3e62.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__7cc2deafa60c41c2a18a184d1caf3e62.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://document-type-container/7cc2deafa60c41c2a18a184d1caf3e62",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__b64743a42354412daa23b3cb89b6535b.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__b64743a42354412daa23b3cb89b6535b.uda
index 2b78c69..4203de1 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__b64743a42354412daa23b3cb89b6535b.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type-container__b64743a42354412daa23b3cb89b6535b.uda
@@ -6,5 +6,5 @@
   "Udi": "umb://document-type-container/b64743a42354412daa23b3cb89b6535b",
   "Dependencies": [],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContainerArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__3cfa74e895834944b5849fdebef1eff2.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__3cfa74e895834944b5849fdebef1eff2.uda
index 07a8aa1..99b2f84 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__3cfa74e895834944b5849fdebef1eff2.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__3cfa74e895834944b5849fdebef1eff2.uda
@@ -46,5 +46,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__77199092b3844af2ac08b5f40d00638a.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__77199092b3844af2ac08b5f40d00638a.uda
index a7a51e2..16f3160 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__77199092b3844af2ac08b5f40d00638a.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__77199092b3844af2ac08b5f40d00638a.uda
@@ -102,5 +102,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__79f8c2bf361c4c3bad27e700b49d9d61.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__79f8c2bf361c4c3bad27e700b49d9d61.uda
index eb7ca6e..9d05710 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__79f8c2bf361c4c3bad27e700b49d9d61.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__79f8c2bf361c4c3bad27e700b49d9d61.uda
@@ -45,5 +45,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__8139143477a34379a84310f0ce4685ea.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__8139143477a34379a84310f0ce4685ea.uda
index c052664..176a11f 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__8139143477a34379a84310f0ce4685ea.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__8139143477a34379a84310f0ce4685ea.uda
@@ -151,5 +151,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__b36551919b1345608bfb3daa8967a11a.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__b36551919b1345608bfb3daa8967a11a.uda
index 899ac92..9c9a653 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__b36551919b1345608bfb3daa8967a11a.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__b36551919b1345608bfb3daa8967a11a.uda
@@ -177,5 +177,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__bbc18d0051d6428d9b75c60f72831166.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__bbc18d0051d6428d9b75c60f72831166.uda
index 57fc8a1..3bef843 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__bbc18d0051d6428d9b75c60f72831166.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__bbc18d0051d6428d9b75c60f72831166.uda
@@ -203,5 +203,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__cf4a5f8bb08d4a9898d69afa4448ac08.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__cf4a5f8bb08d4a9898d69afa4448ac08.uda
index a349698..fda9356 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__cf4a5f8bb08d4a9898d69afa4448ac08.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__cf4a5f8bb08d4a9898d69afa4448ac08.uda
@@ -88,5 +88,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__e643b6f0f5604032bb45726ee0193269.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__e643b6f0f5604032bb45726ee0193269.uda
index 55f43a2..0fe5fe8 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__e643b6f0f5604032bb45726ee0193269.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/document-type__e643b6f0f5604032bb45726ee0193269.uda
@@ -39,5 +39,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.ContentType.DocumentTypeArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c3b98240086f46d49cc939d36e357407.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c3b98240086f46d49cc939d36e357407.uda
index 21e9ea7..cc2efef 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c3b98240086f46d49cc939d36e357407.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c3b98240086f46d49cc939d36e357407.uda
@@ -12,5 +12,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c8079226d6524d9185878213f1f9d491.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c8079226d6524d9185878213f1f9d491.uda
index 9ff57c0..eba6ce8 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c8079226d6524d9185878213f1f9d491.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__c8079226d6524d9185878213f1f9d491.uda
@@ -12,5 +12,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__d9e58962cc7d4599a4219ca9d558f9b3.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__d9e58962cc7d4599a4219ca9d558f9b3.uda
index b6ba7b9..f58a10d 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__d9e58962cc7d4599a4219ca9d558f9b3.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__d9e58962cc7d4599a4219ca9d558f9b3.uda
@@ -12,5 +12,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
diff --git a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__eb8a1f9abc1d43fb8c3f4660f7b63c12.uda b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__eb8a1f9abc1d43fb8c3f4660f7b63c12.uda
index 4d6f561..b2ff4be 100644
--- a/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__eb8a1f9abc1d43fb8c3f4660f7b63c12.uda
+++ b/src/Mikkelhm.Web/umbraco/Deploy/Revision/template__eb8a1f9abc1d43fb8c3f4660f7b63c12.uda
@@ -12,5 +12,5 @@
     }
   ],
   "__type": "Umbraco.Deploy.Infrastructure,Umbraco.Deploy.Infrastructure.Artifacts.TemplateArtifact",
-  "__version": "12.0.2"
+  "__version": "12.0.5"
 }
\ No newline at end of file
